packages/x-charts/src/ChartsLegend/chartsLegend.types.ts

```diff
@@ -1,6 +1,8 @@
 import { ChartsTextStyle } from '../ChartsText';
+import { PieItemId } from '../models';
+import { SeriesId } from '../models/seriesType/common';
 
-export interface LegendItemParams {
+interface LegendItemContextBase {
```

Comment:
Context feels a bit weird, since there is the React Context

`LegendItemCommonParams` or `LegendItemCommonConfig`

Reply:
I used context because of this: https://www.notion.so/mui-org/engineering-Code-Guidelines-X-39105ea968754d989af47346fc3bbc3f?pvs=4#c98e2fe257a1490aa0ba5cd612db1d07 😅

Reply:
How yes, my bad, it's because it's mixed with the other params/config typing so I did not realised it's for a callback :+1: